packages/core/core/src/ConfigResolver.js

```diff
@@ -1,11 +1,11 @@
 // @flow
 import type {FilePath, ParcelConfig, PackageName} from '@parcel/types';
-import {resolveConfig} from '@parcel/utils/config';
+import {resolveConfig} from '@parcel/utils/lib/config';
```

Comment:
Could you change `lib` to `src` for both? We've been preferring to use babel's register instead of building for now.

Some context: there are probably other spots where we're using lib, but it's just something we missed in a PR, and there's a larger PR (#2632) that changes them all back. We would need to change it back when we're ready to release, but we also might not be using deep imports for utils by then. It helps right now because when we pulled out the utils into their own package and used a index file to export everything, there were circular dependencies, but eventually it would be good to untangle things. We are also working on a `@parcel/register` which could look up other fields besides "main" in package.json, so we could configure "main" to point at "lib" and "source" to point at "src" and when we're using `@parcel/register` in development we don't need to worry about building before trying something out. That wouldn't really work for deep imports though, so that's one motivation to not use them.

Reply:
Makes sense to me. Thanks for the details :)